pandas/_libs/tslibs/offsets.pyx

```diff
@@ -791,6 +791,10 @@ cdef class Tick(SingleConstructorOffset):
     def is_anchored(self) -> bool:
         return False
 
+    # This is identical to DateOffset.__hash__, but has to be redefined here
+    # for Python 3, because we've redefined __eq__.
+    def __hash__(self) -> int:
+        return hash(self._params)
```

Comment:
Can you add an empty line below this?

Reply:
Thx

Reply:
Sorry copied it around and then missed this :(